tests/modules/sightings/resources/utils.py

```diff
@@ -78,7 +78,7 @@ def read_sighting(flask_app_client, user, sight_guid, expected_status_code=200):
         'sightings:read',
         f'{PATH}{sight_guid}',
         expected_status_code,
-        response_200={'id'},
+        response_200={'id', 'creator', 'encounters', 'stage', 'assets'},
```

Comment:
Only a limited set of the fields. Do you think a fuller set should be here?

Reply:
This is fine for now. Not a big deal to add more if they are desired later.